src/Cake.Common.Tests/Unit/Tools/OctopusDeploy/OctoPushTests.cs

```diff
@@ -0,0 +1,331 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
```

Comment:
@agc93 we can lose some usings here:

![image](https://cloud.githubusercontent.com/assets/1271146/16129998/29444ab4-33ff-11e6-818b-4f715fc1d99d.png)